test/runtime/sungeun/chpl-env-gen.precomp

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
fair enough. CHPL_HOME should always be defined when running under the testing system 

Maybe just leave it as `chpl_home = os.getenv('CHPL_HOME')`

that way chpl_home will be `None` if $CHPL_HOME isn't set and it'll bark early on rather than trying to run a non-existent printchplenv